app/views/events/participants.html.erb

```diff
@@ -36,6 +36,6 @@
   </table>
   
   
-  <%= link_to t('.back', :default => t("helpers.links.back")),
+  <%= link_to t('.back', :default => t('helpers.links.back')),
```

Comment:
This is almost the only place you use single quotes. I'm not really familiar with what Ruby expects, but wound't it make sense to have everything the same way at least? From what I heard, Ruby prefers single quotes over double quotes whenever possible.